src/Hecke.jl

```diff
@@ -53,7 +53,7 @@ import AbstractAlgebra:
   set_assertion_level,
   set_verbosity_level
 
-import AbstractAlgebra: Solve, coprime_base_steel
+import AbstractAlgebra: Solve, coprime_base_steel, vector_space_dim
```

Comment:
```suggestion
import AbstractAlgebra: Solve, coprime_base_steel
```
this is not needed as Hecke imports everything that gets exported from AA